src/main.rs

```diff
@@ -98,8 +98,12 @@ fn main() -> Result<(), Box<dyn Error>> {
         }
     };
     let path = matches.opt_str("output");
-    let commits_path = &matches.free[1];
-    let commits = fs::read_to_string(commits_path).unwrap();
+    let commits = std::process::Command::new("git")
+        .arg("log")
+        .arg("--numstat")
+        .output()
+        .expect("This is not a Git repository");
+    let commits = String::from_utf8(commits.stdout).unwrap();
```

Comment:
Now, `git log --numstat` is called *from the application itself*.  This not only fixes #16 due to always ensuring the correct format but also replaces the reading from `stdin`, see #3, since one obviously does not need to do anything to get the history extracted.

Reply:
Shall I make it literally an option?

Reply:
I think you replied to the wrong comment here? This against letting the program call `git log --numstat` itself, in favor for stdin reading/parsing.

Reply:
:wrench: :heavy_check_mark: